app/api/shared.js

```diff
@@ -1,9 +1,18 @@
 const { createBearerHeader } = require('../auth/jwt-utils')
-const addHeaders = (user, audience = 'aphw-ddi-api') => {
+const { api } = require('../constants/audience')
+
+const addHeaders = (user, audience = api) => {
+  const userAgent = {}
+
+  if (user.userAgent) {
```

Comment:
Do we need to check if user is null by coalescing ie.. user?.userAgent - just an inconsistency with say lines 12 + 13 below. Happy to leave though if you think ot an issue

Reply:
yeah, think we do, updated.